### PR TITLE
feat: Switch the llm to ChatVertexAI

### PR DESCRIPTION
**Pull request by anubhav756**

### CODE REVIEW

File: llm_demo/orchestrator/langgraph/react_graph.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ async def create_graph(
     tool_node = ToolNode(tools)
 
     # model node
-    model = VertexAI(max_output_tokens=512, model_name=model_name, temperature=0.0)
+    model = ChatVertexAI(max_output_tokens=512, model_name=model_name, temperature=0.0)
```
Comment (Yuan325) [inline llm_demo/orchestrator/langgraph/react_graph.py:88]: In order to use tool calling, we will have to add `.bind_tools(tools)`. source: https://python.langchain.com/docs/how_to/tool_calling/

Comment (anubhav756) [reply llm_demo/orchestrator/langgraph/react_graph.py:88]: I've added the `.bind_tools(tools)` call for now. We can add other required changes in a future PR.

**Review by kurtisvg — APPROVED** [review]

**Pull request merged**